### ISSUE TITLE
Close button not showing

### ISSUE THREAD

**Issue by pbenkoe** [MailOnline/ImageViewer#117]
I want to show a single image with the code below, but the close button is not showing up. The rest is working perfectly though, image shows up and i can dismiss with the swipe gesture...

```
let vc = GalleryViewController(startIndex: 0,
                               itemsDatasource: PhotoProvider(image: viewModel?.userViewModel.availableProfilePhoto()),
                               configuration: [GalleryConfigurationItem.closeButtonMode(.builtIn),
                                               GalleryConfigurationItem.thumbnailsButtonMode(.none)])
presentImageGallery(vc)

**Comment by juarezjaramillo** [comment]
I have the same problem, and the configuration `GalleryConfigurationItem.toggleDecorationViewsBySingleTap(true),` no longer exists.

**Comment by coreywiley** [comment]
See [94](https://github.com/MailOnline/ImageViewer/issues/94). Pod needs updating. Maybe open an issue to update the pod?

**Comment by dars** [comment]
I have the same problem close button not showing, and use Carthage to install imageView not cocoapods (ver 4,0,0)
GalleryConfigurationItem swipeToDismissMode and toggleDecorationViewsBySingleTap have error showing has no member
please help me..

**Comment by dars** [comment]
finally work around find the answer for now
when use Carthage to install, it's still download ver3.0
so I download latest file from GitHub and replace Carthage/Checkouts/ImageViewers folder files
then run carthage build ImageViewer (do not use carthage update)
it's fix the probleam, everything looks fine

**Comment by zfoltin** [comment]
Hopefully this will be resolved with the 4.1.0 release
